specs/tabs/tabs.md

```diff
@@ -0,0 +1,173 @@
+# fast-tabs
+
+## Overview
+
+*Tabs* are a set of layered sections of content, known as tab panels, that display one panel of content at a time. Each tab panel has an associated tab element, that when activated, displays the panel. The list of tab elements is arranged along one edge of the currently displayed panel.
+
+### Use Cases
+
+- Steve is researching for an upcoming vacation for him and his family. He opens his web browser and opens a new tab to make a rental car reservation and another for his flight information. Steve can quickly switch back and fourth between tabs to get the information he needs for the rental car reservation.
+
+- Monika visits a website to learn how to fix her leaky sink. The website displays a vertical tab interface with each step. Monika can jump ahead steps or go back to previous steps by selecting various tabs.
+  
+### Features
+
+- **Orientation:** Allows the tab list to be oriented vertically to the left or right (depending on language region) of the tab content or horizontally above the content.
+- **Tab activation:** Offers a way to activate a tab and it's content by either focusing a tab or by key press (Space or Enter).
+- **Supplemental content:** Offers a way to add content to the left and/or right of the the tab list.
+
+### Risks and Challenges
+
+*Tabs* has specific guidance on DOM structure by the [WC3](https://w3c.github.io/aria-practices/examples/tabs/tabs-2/tabs.html). This structure lacks logical groupings by separating the tab from its content. Some component libraries compensate for this by creating some kind of intermediary grouping that makes it easier for app authors to implement. Even though the DOM structure disassociates that logical grouping. While other component libraries stick to the DOM structure model. 
+
+### Prior Art/Examples
+- [FAST-DNA Pivot (React)](https://explore.fast.design/components/pivot)
+- [Material UI](https://material-ui.com/components/tabs/)
+- [Lightning Design](https://www.lightningdesignsystem.com/components/tabs/)
+- [Ant Design](https://ant.design/components/tabs/)
+- [Atlassian](https://atlaskit.atlassian.com/packages/core/tabs)
+- [Windows (UWP)](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tab-view)
+
+---
+
+### API
+
+*Component name:*
+- `fast-tabs`
+
+*Attributes:*
+- `orientation` - enum
+  - horizontal - default
+  - vertical
+- `activeId` - string
```

Comment:
For this, I think it might be better to have this be a reference to the tab element that is active, rather than an id, if possible.

Reply:
`activeTab`?

Reply:
That works for me.

Reply:
I'm struggling to see why a reference to the active tab is better than an ID? how would an app author set the `activeTab`? With `activeId` it would be something like this:

```HTML
<fast-tabs activeId="tab2">
    <fast-tab-item id="tab1">
        <span slot="tab">Tab One</span>
        Content of the first tab
    </fast-tab-item>
    <fast-tab-item id="tab2">
        <span slot="tab">Tab Two</span>
        Content of the Second tab
    </fast-tab-item>
    <fast-tab-item id="tab3">
        <span slot="tab">Tab Three</span>
        Content of the Third tab
    </fast-tab-item>
</fast-tabs>
```

Reply:
If the customer is setting the component up in JS, this allows them to create the tab items and set the active item without needing to set up an id. This also enables templating libraries that support the "ref" concept to let their users assign the active tab using a ref.

So, those are a couple of use cases. Beyond that, for some of our other web components, we've been following a pattern of using a control ref property with a corresponding id property. So, I'd like to do this for consistency both with that and the native platform, which uses the same pattern for controls like label.